app/views/mailer/mfa_recommendation_announcement.html.erb

```diff
@@ -0,0 +1,126 @@
+<% @title = "Enable multi-factor authentication (MFA) on your RubyGems account" %>
+<% @sub_title = "👋 Hi #{@user.handle}" %>
+
+<!-- Body -->
+<table width="100%" border="0" cellspacing="0" cellpadding="0" bgcolor="#ffffff">
+  <tr>
+    <td class="content-spacing" style="font-size:0pt; line-height:0pt; text-align:left" width="20"></td>
+    <td>
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="35" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="40" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <div class="h3-1-center" style="color:#1e1e1e; font-family:Georgia, serif; min-width:auto !important; font-size:20px; line-height:26px;">
+        <p>
+          Today, we've announced our security-focused ambitions to the community. 
+          Attacks on the software supply chain are increasing and the second most common attack is account takeovers.
+        </p>
+        <br/>
+
+        <p>
+          On August 15, 2022, we will begin requiring maintainers of packages with more than 180 million downloads to have multi-factor authentication (MFA) enabled.
+          Furthermore, we are officially recommending that maintainers of packages with more than 165 million downloads enable MFA. 
+          This recommendation will also be shown by the gem client. 
```

Comment:
Heya Aditya! With more thought, I think it'd be safer for us to stick with `165M` (for MFA recommended) and `180M` (for MFA required). I had discussed this with @jenshenny earlier this week, this is our rationale:

* A lot of the existing documentation has consistently referenced 165M/180M as our thresholds. The documentation referred to here includes references in the codebase, issues, PRs, and the original RFC (per [Policy Rollout](https://github.com/rubygems/rfcs/pull/36/files#diff-3d5cc3acc06fe7e9150fdbfc43399c5ad42572c122187774bfc3a4857df524f1R98)). I'm sensitive to the fact that there may have been interested parties that have followed those artefacts; they would be surprised about the new 150M/165M thresholds.
* The strategy behind creating a 15M buffer between `MFA recommended` and `MFA required` along with a 2 month time grace period was to provide maintainers time to adapt to the new policy. If we change the thresholds to 150/165M on Phase 3 launch day, users who are close to the 165M download threshold would have limited time to act.
* Additionally, we'll begin surfacing warnings to maintainers with 150M-165M downloads which is unexpected since the blog post notes 165M+. 

Instead of revising the original thresholds (165M/185M), I think the safer approach is updating the blog post with clearer wording. With that said, I've gone ahead and dropped the commit that revised the thresholds to 150M/165M and reverted back to the original amounts.

Reply:
sure. makes sense. We can continue with recommendation only for 165m downloads.